src/deploy/standalone/logrotate_noobaa.conf

```diff
@@ -9,8 +9,7 @@
     create 644 root root
     sharedscripts
     postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+        kill -HUP $(pidof rsyslogd) >/dev/null 2>&1 || true
```

Comment:
1. Do you know why is it needed to redirect the output of the kill command to /dev/null ? what happens if we just leave the output as is?

2. Redhat article suggests for RHEL8  and above to use systemd to send this signal to the rsyslog.service as described here - https://access.redhat.com/solutions/1294. However we might be a bit more portable by using the suggested killall option as described in this rsyslog issue - https://github.com/rsyslog/rsyslog/issues/3952

```
postrotate 
    killall -HUP rsyslogd || true
endscript
```

Reply:
@guymguym code you suggested is is working.